invenio/base/static/css/invenio.css

```diff
@@ -1,28 +1,30 @@
 /*
-* -*- mode: text; coding: utf-8; -*-
+ * -*- mode: text; coding: utf-8; -*-
+ *
+ * This file is part of Invenio.
+ * Copyright (C) 2009, 2010, 2011, 2012, 2013, 2015 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
 
-   This file is part of Invenio.
-   Copyright (C) 2009, 2010, 2011, 2012, 2013, 2015 CERN.
-
-   Invenio is free software; you can redistribute it and/or
-   modify it under the terms of the GNU General Public License as
-   published by the Free Software Foundation; either version 2 of the
-   License, or (at your option) any later version.
-
-   Invenio is distributed in the hope that it will be useful, but
-   WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-   General Public License for more details.
-
-   You should have received a copy of the GNU General Public License
-   along with Invenio; if not, write to the Free Software Foundation, Inc.,
-   59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
-*/
 body {
    color: #000;
    background: #fff;
    margin: 0px;
    padding: 0px;
+   position: relative;
```

Comment:
this file is going to be removed in #2047. where do you need to change this css to be change?